iotdb-core/datanode/src/main/java/org/apache/iotdb/db/service/DataNodeShutdownHook.java

```diff
@@ -90,6 +95,23 @@ private void startWatcher() {
   @Override
   public void run() {
     logger.info("DataNode exiting...");
+    if (CommonDescriptor.getInstance().getConfig().isEnableAuditLog()) {
+      AuditLogFields fields =
+          new AuditLogFields(
+              "root",
+              -1,
+              null,
+              AuditEventType.DESTROY_KEY,
+              AuditLogOperation.CONTROL,
+              PrivilegeType.SECURITY,
+              true,
+              null,
+              null);
+      String logMessage =
+          String.format("Successfully destroy the tsfile encrypt key in DataNode %s", nodeLocation);
```

Comment:
U sure the tsfile encrypt key is destroyed here? Better record this audit log just behind the code execution.